tests/testthat/tests-date_axis.R

```diff
@@ -21,22 +21,27 @@ test_that("axis ticks in the right location", {
 
 test_that("axis labels centered on interval", {
   gs <- gsplot() %>%
-    points(seq(as.Date("2013-01-01"), as.Date("2013-12-31"), "days"), 1:31) %>%
+    points(seq(as.Date("2013-01-01"), as.Date("2013-12-31"), "days"), 1:365) %>%
     date_axis(side=1, lab.pos = "interval", tick.int="month", "snap.to"="month")
   
-  # labels <- lazyeval::lazy_eval(gs$side.1[[2]]$at, data=list(object=gs))
-  # expect_true(all.equal(labels[1], as.Date("2013-01-15"), tolerance=0.01))
-  # expect_true(all.equal(labels[7], as.Date("2013-07-15"), tolerance=0.01))
-  # expect_true(all.equal(labels[12], as.Date("2013-12-15"), tolerance=0.01))
+  second.axis <- gs$side.1
+  which.axis <- which(names(second.axis)== 'axis')
+  labels <- lazyeval::lazy_eval(gs$side.1[[which.axis[2]]]$at, data=list(object=gs))
```

Comment:
@jiwalker-usgs I had to add this calculation (which.axis) because the order got jiggered around. It doesn't seem like that would affect your code...but please double check that things are working.